libs/affinity/include/hpx/affinity/detail/partlit.hpp

```diff
@@ -194,7 +194,7 @@ namespace hpx { namespace threads { namespace detail {
 
     private:
         // silence MSVC warning C4512: assignment operator could not be generated
-        partial_literal_string& operator=(partial_literal_string const&);
+        partial_literal_string& operator=(partial_literal_string const&) = default;
```

Comment:
This might have to be `=delete` if anything. I'll investigate.

Reply:
I removed this declaration altogether as recent versions of the compiler don't generate the bogus warning anymore.

Reply:
Thank you!